tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -511,14 +510,17 @@ public:
    ////////////////////////////////////////////////////////////////////////////
    /// \brief Creates a custom column with a value dependent on the processing slot and the current entry.
    /// \param[in] name The name of the custom column.
-   /// \param[in] expression Function, lambda expression, functor class or any other callable object producing the temporary value. Returns the value that will be assigned to the custom column.
+   /// \param[in] expression Function, lambda expression, functor class or any other callable object producing the
+   /// temporary value. Returns the value that will be assigned to the custom column.
    /// \param[in] columns Names of the columns/branches in input to the producer function (excluding slot and entry).
```

Comment:
This is clang-format messing up our docs

Reply:
I thought I protected all code parts which need to be left alone by clang-format. I will take care.